### PR TITLE
Added support for imagePullSecrets in Loki Helm chart

### PR DESCRIPTION
**Pull request by lukipro**
**What this PR does / why we need it**:
This PR adds support for imagePullSecrets in the Loki Helm chart. This is required if the extra container image is hosted in a local registry.

Code is taken from stable/grafana chart.

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: production/helm/loki-stack/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: "v1"
 name: loki-stack
-version: 0.30.0
+version: 0.31.0
```
Comment (cyriltovena) [inline production/helm/loki-stack/Chart.yaml]:
```suggestion
version: 0.31.1
```

File: production/helm/loki/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: "v1"
 name: loki
-version: 0.25.0
+version: 0.26.0
```
Comment (cyriltovena) [inline production/helm/loki/Chart.yaml]:
```suggestion
version: 0.26.1
```

File: production/helm/loki-stack/Chart.yaml (hunk 1 above)
Comment (lukipro) [reply production/helm/loki-stack/Chart.yaml]: You have a point there, will update accordingly.

File: production/helm/loki/Chart.yaml (hunk 2 above)
Comment (lukipro) [reply production/helm/loki/Chart.yaml]: You have a point there, will update accordingly.

**Pull request merged**